cloud/store/service.go

```diff
@@ -60,6 +60,9 @@ func (s *Service) SchemaReader() *schema {
 }
 
 func (s *Service) AddClass(cls *models.Class, ss *sharding.State) error {
+	if cls == nil || cls.Class == "" {
```

Comment:
does `sharding.State` also need to be validated ?

Reply:
That's fine since we need to verify only the class because we are referencing it inside the function.